packages/react/src/playground/index.js

```diff
@@ -357,9 +357,7 @@ class Playground extends React.Component {
     this.state.topNavSearchQueryValue &&
     this.state.topNavSearchQueryValue.length > 0
       ? autocompleteSuggestionOptions.filter(option =>
-          option.label
-            .toLowerCase()
-            .startsWith(this.state.topNavSearchQueryValue.toLowerCase())
+          option.value.match(new RegExp(this.state.topNavSearchQueryValue, "i"))
```

Comment:
It's unfortunate that there isn't a straightforward to get the text content of a node. airbnb/enzyme makes this available but does it by recursing through the text nodes of an element.

Limitations here may be a dealbreaker, but I'm curious for some thoughts.